langchain/embeddings/spacy_embeddings.py

```diff
@@ -0,0 +1,110 @@
+import importlib.util
+from typing import Any, Dict, List, Optional
+from pydantic import BaseModel, Extra, root_validator
+from langchain.embeddings.base import Embeddings
+
+
+class SpacyEmbeddings(BaseModel, Embeddings):
```

Comment:
lets expose this in __init__ of embeddings module as well

Reply:
> lets expose this in **init** of embeddings module as well

done. exposed in init.